src/EVault/shared/Errors.sol

```diff
@@ -26,6 +26,7 @@ contract Errors {
     error E_ControllerDisabled();
     error E_CollateralDisabled();
     error E_ViolatorLiquidityDeferred();
+    error E_SameBlockLiquidation();
```

Comment:
If we make it configurable, let's change this error message. We need a name for this, maybe `E_LiquidationCoolOffPeriod` ?